src/components/select2-view.js

```diff
@@ -249,6 +249,18 @@ export default View.extend({
         this.$select.attr('disabled', true)
       }
     })
+
```

Comment:
@EzeKoren no estoy seguro de esto.

Este componente se usa en varios lugares.
value podría ser null o empty en algún lugar y esta bien así. 

Que sea null o empty no quiere decir que deba tener algo seleccionado por defecto.

Podríamos romper algo.
Si queremos dejarlo acá este comportamiento debería ser opcional desde la creación del input

por ejemplo para que solo se active cuando el combo se usa desde el form de creación de tareas.

debería ser una opción como por ejemplo this.autoselectSingleOption

Reply:
Para que se entienda, lo malo de dejar acá este comportamiento es que también se activa en la edición.

no queremos que esto suceda si el combo no tiene nada seleccionado, y eso podría cambiarlo a la fuerza y no te darías cuenta

Reply:
Listo. Puse una flag para habilitar este comportamiento. Al crear el select2, hay que setear la prop `autoselectSingleOption` como `true`